Assets/_Application/Layers/LayerTypes/CartesianTiles/CartesianTileSubObjectColorLayerGameObject.cs

```diff
@@ -103,26 +101,24 @@ private List<CartesianTileSubObjectColorLayerGameObject> GetFlatHierarchy(LayerD
         {
             var list = new List<CartesianTileSubObjectColorLayerGameObject>();
 
-            AddLayersRecursive(root, list);
+            //TODO broken feature fix this!
```

Comment:
Moet dit niet nog aangepakt?

Reply:
ja dat is in main ook stuk, daar is ticket 1869 voor